package.json

```diff
@@ -142,20 +144,34 @@
   },
   "homepage": "https://threejs.org/",
   "dependencies": {
+    "@codemirror/commands": "^6.1.0",
+    "@codemirror/lang-javascript": "^6.0.2",
+    "@codemirror/language": "^6.2.1",
+    "@codemirror/lint": "^6.0.0",
+    "@codemirror/matchbrackets": "^0.19.4",
+    "@codemirror/state": "^6.1.1",
+    "@codemirror/theme-one-dark": "^6.0.0",
+    "@codemirror/view": "^6.2.1",
+    "@lezer/highlight": "^1.0.0",
     "acorn": "1.0.1",
-    "codemirror": "5.65.2",
-    "esprima": "3.1.3",
-    "jsonlint": "1.6.3",
+    "codemirror": "^6.0.1",
+    "eslint4b-prebuilt": "^6.7.2",
+    "esprima": "^3.1.3",
+    "i": "^0.3.7",
+    "jsonlint": "^1.6.3",
+    "npm": "^8.17.0",
+    "rollup-plugin-external-globals": "^0.6.1",
     "signals": "1.0.0",
-    "ternjs": "0.22.0"
+    "ternjs": "0.22.0",
+    "watch": "^1.0.2"
```

Comment:
Are all of these new dependencies are really necessary? 
`watch` `rollup-plugin-external-globals` `npm` `i` `eslint4b-prebuilt`
I didn't find any places where we use them. 
If this is something that is required only for development, we should consider moving them to devDependencies

Reply:
Will check rollup-plugin probably just forget to delete it, but eslint4b-prebuilt is using for linting in codemirror

Reply:
Please clean up unnecessary dependencies